server/controllers/auctions.js

```diff
@@ -36,6 +36,17 @@ router.post('/', (req, res) => {
   });
 });
 
+router.post('/ongoing', (req, res) => {
+  const { user } = req.body;
+  model.getUserBiddingAuctions(user)
+  .then((auctions) => {
+    res.status(200).json(auctions);
+  })
+  .catch((err) => {
+    res.status(500).send(err);
```

Comment:
After thinking about it more, if the we know that the request userId is valid it should probably be a 404 regardless of where the invalid id originated.